src/lib/transition_frontier_persistence/rpc_worker.ml

```diff
@@ -89,12 +89,22 @@ module Make (Inputs : Intf.Worker_inputs) = struct
 
   type t = {connection: Rpc_worker.Connection.t; process: Process.t}
 
-  let create ~directory_name =
+  let create ~directory_name ~logger =
```

Comment:
This doesn't get tracking?

Reply:
I didn't want to add it to the termination set, since I don't believe it's as crucial as the verifier or prover.

Reply:
if the persistence process terminates, it'll get logged, but the daemon will continue